src/styles/_stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+  }
+
+  &--4 > &__star:nth-of-type(-n + 4) {
```

Comment:
You should use the entire stars styling from the previous task, to keep this block interactive